deploy-board/deploy_board/templates/deploys/deploy_history.tmpl

```diff
@@ -63,6 +64,7 @@
                 {% endif %}
             </td>
             <td>{{ deploy_summary.deploy.operator }}</td>
+            <td><a href="/clouds/accounts/{{ deploy_summary.account.provider }}/{{ deploy_summary.account.cell }}/{{ deploy_summary.account.id }}">{{ deploy_summary.account.id }}</a></td>
```

Comment:
Let's hide this link if account is `None` for backward compatibility in cases when Deploy Board will be deployed before cluster management service.

Or if deployment was done in the cluster which was provisioned manually.